locksmith/src/controllers/v2/ticketsController.ts

```diff
@@ -24,4 +24,26 @@ export class TicketsController {
     )
     response.status(200).send({ payload, signature })
   }
+
+  async markTicketAsCheckIn(request: Request, response: Response) {
+    const lockAddress = Normalizer.ethereumAddress(request.params.lockAddress)
+    const network = Number(request.params.network)
+    const metadata = request.body.message.KeyMetaData
```

Comment:
Let's not actually have `KeyMetaData` be submitted since we know the format. Let's just built it here. This way we avoid malicious behaviors.